### DISCUSSION TITLE
where is `revert`

### DISCUSSION THREAD

**Discussion by lnxfsf** [jesseduffield/lazygit#3043] [Q&A] [answered]
There is only undo last action, on `z`.
But it's doing hard reset.

But is there option for `revert`.

If you could make this, that would be great !

**Comment by mark2185** [comment, marked as answer]
You can revert a specific commit by going to the `Commits` panel and pressing `t`, I believe.

Is that what you are looking for?

**Reply by jasonribble** [reply]
It would be nice if this was on the helper menu

**Reply by stefanhaller** [reply]
Not sure what you mean by "helper menu", but if you mean the keybindings menu, it's in there: 
<img width="1044" height="812" alt="CleanShot 2026-09-03 at 09 16 02@2x" src="https://github.com/user-attachments/assets/ed013331-7d31-4640-ad52-d03f9d1b091d" />

**Reply by jasonribble** [reply]
Hey @stefanhaller thanks! I mean directly here

<img width="1041" height="505" alt="image" src="https://github.com/user-attachments/assets/a16077d7-caa5-40bf-a527-1cdda1d71987" />


But now, I see that there's a `?` to open up the keybindings menu. Thank you!

**Reply by stefanhaller** [reply]
That one-line bar down there has limited space, and is already too crowded with commands that don't fit on a smaller screen. Adding it there is just not an option unless we remove others. Now, what are the most important commands to show there is probably a matter of personal opinion and can be discussed forever...

**Reply by jasonribble** [reply]
> probably a matter of personal opinion and can be discussed forever...

Totally agreed. That makes me wonder if it is configurable?

**Reply by stefanhaller** [reply]
No it's not, and I don't think it needs to be.

**Reply by jasonribble** [reply]
Thanks mate. Great software
